imagetool/src/main/bin/imagetool.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
You sure you don't want /bin/sh?  I grepped through deploy tooling and you always chose #!/bin/sh and I think that's probably slightly better.

Reply:
/bin/sh is not available on Ubuntu, only dash or bash.  I think /bin/bash is the better option, as bash seems to be the new common denominator.